Getting Started With CSS/index.md

```diff
@@ -0,0 +1,152 @@
+# GETTING STARTED WITH CSS
+
+CSS is an abbreviation for cascading style sheets. It is a style language used in expressing the display of a document  written in a markup language like HTML. Alongside HTML and Javascript, CSS is a foundation technology in the worldwide web.
+## Functions of CSS
+CSS aids users carry out the presentation of web pages and adjust their presentation to different types of gadgets such as large screens or small screens. CSS also helps users apply elements consistently throughout the site pages such that lists and headings look alike throughout the site pages.
+## CSS Properties and How to Use Them
+### 1.Changing The Color of A Text
+It is done by merely adding the suitable CSS selector and specifying the color property and value wanted. Users can use CSS selectors to style elements by adding the **style** element to your code like this;
+```CSS
+<style>
+ h2{
+     color: red;
+ }
+ </style>
+ ```
+ This will change all the header2 to the color red.
+ The CSS selector can be incorporated in HTML like this;
```

Comment:
consider changing this to
```suggestion
This will change all the h2 elements to the color red.

 Although not advisable in practice, you can also use inline styles to achieve the same results as illustrated in the example below.
```